### DISCUSSION TITLE
Not connecting to port 13378 from desktop or iOS app using Tailscale

### DISCUSSION THREAD

**Discussion by autryld** [advplyr/audiobookshelf#1981] [Q&A]
I am able to connect to the audiobookshelf service on my Linux server but not using Tailscale.  However, I **can** connect to Plex via its tailscaleIP:32400 from my desktop and iPhone app.  That makes me think it's not a Tailscale configuration issue.  If there's something I need to check, please suggest it. 

The message I get on the iPhone app is "❗ Failed to ping server".

Thanks for any advice!

**Comment by autryld** [comment]
Five weeks have passed since my original post with no success (I didn’t really try very hard). I was finally able to connect to the server using its local LAN address instead the TailscaleIP address. There’s no mistaking this success as a fluke because I was several miles away from home at the time. It would be nice to understand why the TailscaleIP does not work.

**Comment by oboe45** [comment]
when this happens on my iOS pad or iMac, its usually because the Tailscale connection has dropped on the device you are trying to use.   Load the Tailscale app, click connect and you should then be able to make the connections.   This seems to happen when the device hasn't been used for a while, or, at leat in may case this is the problem and the cure
